### ISSUE TITLE
With Sveltekit, component is displayed after page load.

### ISSUE THREAD

**Issue by AxelBriche** [material-components/material-web#4024]
With Sveltekit, components are displayed with delay and create a "conent flash".
Server side is not supported?

https://user-images.githubusercontent.com/1295251/222897510-e508b560-0f5a-4be0-86d9-3273991e693d.mp4

**Comment by asyncLiz** [comment]
I don't believe we'll have capacity to help debug Sveltekit issues. These components are currently not SSR-tested, so support may be spotty.

Do you have specifics on how Sveltekit is expecting web components with shadow DOM to be SSR-compatible?
